egs/wsj/s5/utils/lang/check_g_properties.pl

```diff
@@ -73,8 +73,8 @@
   }
 }
 
-close(O);  # tell 'fstcompile | fstinfo' pipeline that its input is done.
-while (<I>) {
+close(I);  # tell 'fstcompile | fstinfo' pipeline that its input is done.
+while (<O>) {
   if (m/cyclic\s+/) {
```

Comment:
OK, I think at this point it should say m/cyclic\s+y/
See if that makes it behave as expected.